inference-engine/tests/ngraph_functions/src/utils/ngraph_helpers.cpp

```diff
@@ -40,8 +40,11 @@ std::ostream &operator<<(std::ostream &os, const ReductionType &m) {
         case LogicalAnd:
             os << "LogicalAnd";
             break;
-        case LogicalXor:
-            os << "LogicalXor";
+        case L1:
+            os << "L1";
+            break;
+        case L2:
+            os << "L2";
```

Comment:
`ReduceL2`

Reply:
Fixed in ff00896e4f32744195fd0f021ebb9b9e4ae2c5cd.